backend/test/model_test/request_institution_children_test.py

```diff
@@ -0,0 +1,257 @@
+            self.other_institution
+        )
+
+        request.send_notification(self.institution.key)
+        
+        create_notification_message.assert_called_with(
+            user_key=self.admin.key, 
+            current_institution_key=self.institution.key,
+            sender_institution_key=self.institution.key,
+            receiver_institution_key=self.other_institution.key
+        )
+
+        send_notification.assert_called_with(
+            current_institution=self.institution.key, 
+            receiver_key=self.other_institution.admin, 
+            notification_type='REQUEST_INSTITUTION_CHILDREN',
+            message='mocked_message'
+        )
+
+
+    @patch.object(Invite, 'send_notification')
+    @patch.object(Invite, 'create_notification_message', return_value='mocked_message')
+    def test_send_response_notification(self, create_notification_message, send_notification):
+        """Test send response notification."""
+        request = generate_request(
+            self.admin, self.institution, 
+            self.other_institution
+        )
+
+        for action in ['ACCEPT', 'REJECT']:
+            request.send_response_notification(
+                current_institution=self.other_institution.key, 
+                invitee_key=self.other_institution.admin,
+                action=action
+            )
+            
+            create_notification_message.assert_called_with(
+                user_key=self.other_institution.admin, 
+                current_institution_key=self.other_institution.key,
+                sender_institution_key=self.other_institution.key,
+                receiver_institution_key=self.institution.key
+            )
+
+            notification_type = 'ACCEPT_INSTITUTION_LINK' if action == 'ACCEPT' else 'REJECT_INSTITUTION_LINK'
+            send_notification.assert_called_with(
+                current_institution=self.other_institution.key, 
+                sender_key=self.other_institution.admin, 
+                receiver_key=self.admin.key, 
+                notification_type=notification_type,
+                message='mocked_message'
+            )
+
+
+    @patch.object(EmailSender, 'send_email')
+    def test_send_email(self, send_email):
+        """Test send email."""
+        request = generate_request(
+            self.admin, self.institution, 
+            self.other_institution
+        )
+
+        email_json = {
+            'institution_parent_name': self.institution.name,
+            'institution_parent_email': self.institution.institutional_email,
+            'institution_child_name': self.other_institution.name,
+            'institution_child_email': self.other_institution.institutional_email,
+            'institution_requested_key': self.other_institution.key.urlsafe()
+        }
+
+        request.send_email(host='somehost')
+        send_email.assert_called_with(email_json)
+
+
+    @patch.object(EmailSender, 'send_email')
```

Comment:
same thing

Reply:
In both cases, the RequestLinkEmailSender call the send_email from EmailSender. So the method must be patched on this class